deployment/7_upgrade_2_1_fix.js

```diff
@@ -0,0 +1,326 @@
+const semver = require("semver");
+const childProcess = require("child_process");
+const MultiSig = require("../build/MultiSigWallet");
+const ModuleRegistry = require("../build/ModuleRegistry");
+const Upgrader = require("../build/UpgraderToVersionManager");
+const DeployManager = require("../utils/deploy-manager.js");
+const MultisigExecutor = require("../utils/multisigexecutor.js");
+
+const ApprovedTransfer = require("../build/ApprovedTransfer");
+const CompoundManager = require("../build/CompoundManager");
+const GuardianManager = require("../build/GuardianManager");
+const LockManager = require("../build/LockManager");
+const NftTransfer = require("../build/NftTransfer");
+const RecoveryManager = require("../build/RecoveryManager");
+const TokenExchanger = require("../build/TokenExchanger");
+const MakerV2Manager = require("../build/MakerV2Manager");
+const TransferManager = require("../build/TransferManager");
+const RelayerManager = require("../build/RelayerManager");
+const VersionManager = require("../build/VersionManager");
+
+const utils = require("../utils/utilities.js");
+
+const TARGET_VERSION = "2.1.0";
```

Comment:
2.2.0

Reply:
Can you also change the `test:deployment` command to include script 7 in the test. That was excluded as 2.2 was planned no-upgrade release.